src/lucky_flow/registry.cr

```diff
@@ -0,0 +1,21 @@
+class LuckyFlow::Registry
+  @@registry = Hash(String, Proc(LuckyFlow::Driver)).new
+  @@running_registry = Hash(String, LuckyFlow::Driver).new
```

Comment:
`@@registry` is for drivers that can be used but aren't made yet,
`@@running_registry` are drivers that have been created and should be used before making a new one